extensions/EXT/EXT_clip_cull_distance.txt

```diff
@@ -273,6 +275,8 @@ Additions to Chapter 7 of the OpenGL ES Shading Language 3.20 Specification
         highp float gl_CullDistance[];
 
     };
+    [[ If OES_shader_io_blocks is supported: ]]
+    If I/O blocks are not in use, these variables are not in gl_PerVertex.
```

Comment:
I think this needs to apply when OES_shader_io_blocks is not supported as well.

Reply:
So drop the `[[ ]]` conditional?

Reply:
Yes, I think so.

Reply:
Done